### PR TITLE
New version: Ablaze.Floorp version 11.10.4

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140948)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Ablaze.Floorp
+PackageVersion: 11.10.4
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.17763.0
+InstallerType: exe
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.installer.yaml:10]:
```suggestion
InstallerType: nullsoft
```

File: manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+PackageLocale: en-US
+Publisher: Ablaze
+PublisherUrl: https://ablaze.one/
+PublisherSupportUrl: https://support.ablaze.one/articles/floorp/
+PrivacyUrl: https://docs.ablaze.one/floorp_privacy_policy/
+Author: Ablaze
+PackageName: Ablaze Floorp
+PackageUrl: https://floorp.app/en/
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://github.com/Floorp-Projects/Floorp/blob/HEAD/LICENSE
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://github.com/Floorp-Projects/Floorp/blob/ESR115/LICENSE
```

File: manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.locale.ja-JP.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- browser
+- cross-platform
+- firefox-fork
+- foss
+- gecko
+- internet-browser
+- open-source
+- web
+- web-browser
+ReleaseNotesUrl: https://blog.ablaze.one/3967/
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.locale.ja-JP.yaml:27]:
```suggestion
ReleaseNotesUrl: https://blog.ablaze.one/4009/
```

File: manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- browser
+- cross-platform
+- firefox-fork
+- foss
+- gecko
+- internet-browser
+- open-source
+- web
+- web-browser
+ReleaseNotesUrl: https://github.com/Floorp-Projects/Floorp/releases/tag/v11.10.4
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.locale.en-US.yaml:28]:
```suggestion
ReleaseNotesUrl: https://blog.ablaze.one/4014/
```

**Review by Hibi-10000 — CHANGES_REQUESTED** [review]

File: manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.installer.yaml (hunk 1 above)
Comment (Exorcism0666) [reply manifests/a/Ablaze/Floorp/11.10.4/Ablaze.Floorp.installer.yaml]: @russellbanks, Komac has replaced "nullsoft" with "exe".

**Review by stephengillie — APPROVED** [review]

**Pull request merged**